src/components/admin/admin-navigation/admin-navigation.scss

```
@@ -1,9 +1,11 @@
 .admin {
     &-logo {
-        height: 150px;
+        margin-top: 2.75rem;
+        margin-left: 2.5rem;
+        margin-bottom: 3.5rem;
```

You can simplify these three declarations into a single shorthand:
```suggestion
        margin: 2.75rem 0 3.5rem 2.5rem;
```

---

Your suggestion was implemented!